app/src/renderer/components/wallet/SendModal.vue

```diff
@@ -90,6 +90,21 @@
         type="between"
       />
     </tm-form-group>
+    <tm-form-group
+      class="action-modal-group"
+      field-id="memo"
+      field-label="Memo"
+    >
+      <span class="input-suffix">
+        {{ (memo ? " - " : "") + "(Sent via Lunie)" }}
```

Comment:
```suggestion
        {{ (memo ? " + " : "") + "(Sent via Lunie)" }}
```

Reply:
there is also a max memo size specified in the genesis file which should be included as part of the form validation (has to include Sent via Lunie too)

Reply:
Are you sure about the plus?

Compare:
`Memo xyz + (Sent via Lunie)`
`Memo xyz - (Sent via Lunie)`

Reply:
no. you are right. i was trying to show that it was appended. we can leave it as a dash.